integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIntrospectVersion.java

```diff
@@ -726,21 +729,34 @@ void testUpdateImageName() {
     String imageUpdate = KIND_REPO != null ? KIND_REPO
         + (WEBLOGIC_IMAGE_NAME + ":" + imageTag).substring(TestConstants.BASE_IMAGES_REPO.length() + 1)
         : TEST_IMAGES_REPO + "/" + WEBLOGIC_IMAGE_NAME_DEFAULT + ":" + imageTag;
-    getLogger().info(" The image name used for update is: {0}", imageUpdate);
+    getLogger().info(" The image name used for the 1st update is: {0}", imageUpdate);
+
+    //1st time update image and restartVersion without image tagging/pushing
```

Comment:
Instead of using the word "tagging/pushing" can we mention Update the image field of Domain Resource with an image that does not exists in the registry.